### PR TITLE
Add article 18 Tableau

### PR DESCRIPTION
**Pull request by ahmadmlafi**
Add article 18 Tableau
Edit tags in article 17

### CODE REVIEW

File: content/post/articles/018_Tableau/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Tableau"
+date: 2021-03-23
+categories: ["مقالات"]
+tags: ["Tableau", "Data Visualization", "Data Science", "Tools"]
+cover:
+    image: images/Tableau.jpg
+alt: "Tableau"
```
Comment (mohnoor94) [inline content/post/articles/018_Tableau/index.md]: please add a tab before these 2 lines

File: content/post/articles/017_Power_BI/index.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: "Power BI"
 date: 2021-03-15
 categories: ["مقالات"]
-tags: ["PowerBI", "DataVisualization", "DataScience", "Tools"]
+tags: ["PowerBI", "Data Visualization", "Data Science", "Tools"]
```
Comment (mohnoor94) [inline content/post/articles/017_Power_BI/index.md:5]: I already updated those so no need to do them again.

**Review by mohnoor94 — CHANGES_REQUESTED** [review]

File: content/post/articles/018_Tableau/index.md (hunk 1 above)
Comment (ahmadmlafi) [reply content/post/articles/018_Tableau/index.md]: Done

File: content/post/articles/017_Power_BI/index.md (hunk 2 above)
Comment (ahmadmlafi) [reply content/post/articles/017_Power_BI/index.md:5]: Ok

Comment (mohnoor94) [reply content/post/articles/017_Power_BI/index.md:5]: Then please undo these changes or open a new PR :)

**Pull request closed without merging**